assets/js/modules/reader-revenue-manager/components/settings/SettingsView.js

```diff
@@ -21,17 +21,32 @@
  */
 import { __ } from '@wordpress/i18n';
 
+/**
+ * Internal dependencies
+ */
+import { useSelect } from '@wordpress/data';
```

Comment:
This import needs moving this to the **WordPress dependencies** section. I'll do it myself to save the round trip.

Reply:
That's a good catch. I have copied that import from some other component.

Reply:
Shouldn't we import it from `googlesitekit-data`?

Reply:
We absolutely should! Thanks for spotting that, I completely missed it :facepalm:

Reply:
I've updated this component, `PublicationOnboardingStateNotice`, and also `ZeroDataNotification`.